BrowserKit/Sources/Common/Theming/DarkTheme.swift

```diff
@@ -130,4 +130,10 @@ private struct DarkColourPalette: ThemeColourPalette {
     var shadowSubtle: UIColor = FXColors.DarkGrey80.withAlphaComponent(0.10)
     var shadowDefault: UIColor = FXColors.DarkGrey80.withAlphaComponent(0.12)
     var shadowStrong: UIColor = FXColors.DarkGrey80.withAlphaComponent(0.16)
+
+    // MARK: - Gradient
+    var vividOrange: UIColor = FXColors.Yellow50
```

Comment:
These color tokens are currently not in our design system Figma - will they be added there?

Reply:
That’s correct. I'm keeping this PR in draft while I’m waiting for @jeffreygee’s confirmation if he can add them and possibly have light and dark variants. Meanwhile, @FilippoZazzeroni suggested that I use the same naming as he did for the shake to summarize feature.

Reply:
@yoanarios Thanks for the review. I will ask UX and come back with a response. The conversation was on Slack.